qutebrowser/mainwindow/tabwidget.py

```diff
@@ -434,11 +434,18 @@ def minimumTabSizeHint(self, index, ellipsis: bool = True):
         Return:
             A QSize of the smallest tab size we can make.
         """
-        text = '\u2026' if ellipsis else self.tabText(index)
+        return self.__minimumTabSizeHintHelper(self.tabText(index),
+                                               self.tabIcon(index), ellipsis)
+
+    @functools.lru_cache(maxsize=100)
+    def __minimumTabSizeHintHelper(self, tab_text: str,
```

Comment:
Please call this `_minimum_tab_size_hint_helper` or so - qutebrowser only uses camel casing for overridden Qt methods, and doesn't use `__foo` methods unless absolutely needed.

Reply:
ah, I confused myself on the naming a bit, sorry :)